README.md

```diff
@@ -82,32 +83,54 @@ More on the [Git workflow for this project](docs/git_workflow.md).
 
 ### Requirements
 
-* PHP 7.4
+* PHP 7.4 (Upgrading to 8.2)
```

Comment:
```suggestion
* PHP >= 8.1
```

Reply:
@jean-gui I think the [composer.json](https://github.com/w3c/w3c-website-frontend/blob/update/readme/composer.json) file also needs updating to reflect 8.1+

Do you want to do this or shall we?

Reply:
It already does